src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiService.scala

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.dsde.rawls.webservice
+
+import org.broadinstitute.dsde.rawls.model._
+import org.broadinstitute.dsde.rawls.openam.UserInfoDirectives
+import org.broadinstitute.dsde.rawls.user.UserService
+import spray.routing.Directive.pimpApply
+import spray.routing._
+
+import scala.concurrent.ExecutionContext
+
+/**
+ * Created by dvoet on 6/4/15.
+ */
+
+trait BillingApiService extends HttpService with PerRequestCreator with UserInfoDirectives {
+  implicit val executionContext: ExecutionContext
+
+  import org.broadinstitute.dsde.rawls.model.UserJsonSupport._
+  import spray.httpx.SprayJsonSupport._
+
+  val userServiceConstructor: UserInfo => UserService
+
+  val billingRoutes = requireUserInfo() { userInfo =>
```

Comment:
user/billing is the user specific billing stuff, i.e. the user's billing accounts or projects. This is for strictly project management apis. Should admin/billing belong here? Possibly